templates/generic_cell_clustering.ipynb

```diff
@@ -628,7 +628,7 @@
     "        xr_channel_names=None,\n",
```

Comment:
<p>This should have erode=True</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/1052/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>